turbo/stages/mock/mock_sentry.go

```diff
@@ -770,11 +770,11 @@ func (ms *MockSentry) insertPoSBlocks(chain *core.ChainPack) error {
 	wr := eth1_chain_reader.NewChainReaderEth1(ms.ChainConfig, direct.NewExecutionClientDirect(ms.Eth1ExecutionService), uint64(time.Hour))
 
 	ctx := context.Background()
-	for i := n; i < chain.Length(); i++ {
-		if err := chain.Blocks[i].HashCheck(); err != nil {
-			return err
-		}
-	}
+	// for i := n; i < chain.Length(); i++ {
```

Comment:
I've commented these out; in order for the execution to flow into el InsertBlocks functionalities, where things like empty auth list or invalid auth sigs can be checked. For tests not-expecting-to-fail, the el should still check all the stuff in HashCheck() after block execution, and float up errors if any.

Reply:
but this messes up block_test.go#wrongtransactionsTrie.json which relies on HashCheck being executed. I'm adding it back